sdk/formrecognizer/Azure.AI.FormRecognizer/src/RawExtractedText.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    public class RawExtractedItem
+    {
+        internal RawExtractedItem() { }
```

Comment:
You use two patterns to initialize the wrapping objects: settable propeties or ctor parameters, I think we should settle on one (I recoment ctor parameters) and use it everywhere.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/10439

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/10439